<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SMSL044" ignored="0" name="Realizar VC" tests="4" failures="2" timestamp="12 jul 2019 19:38:58 GMT" time="34.568" errors="0">
  <testcase name="@BeforeTest setUpClass" time="0.066" classname="components.HookMobile">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at log.LogMobile.intilizedLogger(LogMobile.java:15)
	at initializer.InitilizeHookMobile.OpenLogger(InitilizeHookMobile.java:118)
	at components.HookMobile.setUpClass(HookMobile.java:28)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest setUpClass -->
  <testcase name="@BeforeMethod setUp" time="0.0" classname="components.HookMobile">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@AfterMethod tearDown" time="0.019" classname="components.Hook">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:657)
	at java.util.ArrayList.get(ArrayList.java:433)
	at helpers.PageHelper.closeTabs(PageHelper.java:149)
	at initializer.InitilizeHook.CloseAllTabs(InitilizeHook.java:74)
	at components.Hook.tearDown(Hook.java:53)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterTest tearDownClass" time="0.0" classname="components.Hook">
    <skipped/>
  </testcase> <!-- @AfterTest tearDownClass -->
  <testcase name="@AfterTest tearDownClass" time="0.0" classname="components.HookMobile">
    <skipped/>
  </testcase> <!-- @AfterTest tearDownClass -->
  <testcase name="S_Home_02" time="0.004" classname="suiteTest.Regresion_RealizarVC">
    <skipped/>
  </testcase> <!-- S_Home_02 -->
  <testcase name="M_Ingresar_01" time="5.582" classname="suiteTest.Regresion_Despacho">
    <failure type="java.lang.ClassCastException" message="pages.D_LoginPage cannot be cast to pages.M_LoginPage">
      <![CDATA[java.lang.ClassCastException: pages.D_LoginPage cannot be cast to pages.M_LoginPage
	at steps.M_EntrarVC_IngresarStep.Run(M_EntrarVC_IngresarStep.java:29)
	at suiteTest.Regresion_Despacho.M_Ingresar_01(Regresion_Despacho.java:16)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- M_Ingresar_01 -->
  <testcase name="S_Home_01" time="30.082" classname="suiteTest.Regresion_Despacho">
    <failure type="java.lang.AssertionError" message="Fallo test - S_Home_01 - Step: 05_Se verifica el mensaje de Invitacion Enviada">
      <![CDATA[java.lang.AssertionError: Fallo test - S_Home_01 - Step: 05_Se verifica el mensaje de Invitacion Enviada
	at steps.D_Mobile_Envio_InvitacionStep.Run(D_Mobile_Envio_InvitacionStep.java:99)
	at suiteTest.Regresion_Despacho.S_Home_01(Regresion_Despacho.java:21)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- S_Home_01 -->
  <testcase name="M_Mobile_Atender_01" time="0.0" classname="suiteTest.Regresion_Despacho">
    <skipped/>
  </testcase> <!-- M_Mobile_Atender_01 -->
</testsuite> <!-- Realizar VC -->
